### PR TITLE
Add web framework salvo

### PR DESCRIPTION
**Pull request by chrislearn**

### CODE REVIEW

**Comment by palfrey** [comment]
This has conflicts with current master README. Can you fix them please?

**Comment by chrislearn** [comment]
fixed

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: rust
```
Comment (palfrey) [inline .travis.yml:1]: This file shouldn't be here!

**Comment by palfrey** [comment]
We've now fixed the lack of alphabetical ordering (always in the `CONTRIBUTING.MD`, but not enforced until recently), and this new entry is now in the wrong place. See build log error

```
--- original
+++ modified
@@ -14,7 +14,7 @@
 oltdaniel/zap — A lightning fast http framework for Rust
 Rocket — Rocket is web framework for Rust (nightly) with a focus on ease-of-use, expressability, and speed build badge
 Rustless — A REST-like API micro-framework inspired by Grape and Hyper build badge
+salvo-rs/salvo — an easy to use webframework base on hyper and tokio. build build
 Saphir — A progressive web framework with low-level control, without the pain.
 tiny-http — Low level HTTP server library build badge
-tomaka/rouille — Web framework in Rust build badge
-salvo-rs/salvo — an easy to use webframework base on hyper and tokio. build build
\ No newline at end of file
+tomaka/rouille — Web framework in Rust build badge
\ No newline at end of file

Error: ErrorMessage { msg: "Sorting error" }
```

**Pull request merged**